pbft/test/pbft_test.cpp

```diff
@@ -225,7 +225,7 @@ namespace bzn::test
         service.apply_operation(op);
     }
 
-    TEST_F(pbft_test, messages_after_high_water_mark_dropped)
+    TEST_F(pbft_test, DISABLED_messages_after_high_water_mark_dropped)
```

Comment:
Should we try to make this test work?

Reply:
See above

Reply:
Shouldn't it still work per the new, very high high water mark? Or be deleted entirely if the HWM is removed.